pkg/kubectl/cmd/config/config.go

```diff
@@ -66,59 +80,323 @@ func NewCmdConfig(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	return cmd
 }
 
-func (o *pathOptions) getStartingConfig() (*clientcmdapi.Config, string, error) {
-	filename := ""
+func NewDefaultPathOptions() *PathOptions {
+	ret := &PathOptions{
+		LocalFile:  ".kubeconfig",
+		GlobalFile: os.Getenv("HOME") + "/.kube/.kubeconfig",
```

Comment:
I was actually going to pitch `.kube/config` to match better with `.git/config` and `.config/openshift/config`, but I'd like to do that in a separate pull.  I need to be able to gracefully handle the migration case and that gets sticky enough to split out.